src/metabase/middleware/security.clj

```diff
@@ -41,11 +54,13 @@
                                  (when config/is-dev?
                                    "localhost:8080")
                                  ;; inline script in index.html that sets `MetabaseBootstrap` and the like
-                                 "'sha256-xlgrBEvjf72cXGba6bCV/PwIVp1DcbdhY74VIXN8fA4='"
+                                 (format "'sha256-%s'" index-bootstrap-js-hash)
                                  ;; Web Font Loader font configuration (WebFontConfig) in index.html
-                                 "'sha256-6xC9z5Dcryu9jbxUZkBJ5yUmSofhJjt7Mbnp/ijPkFs='"
+                                 (format "'sha256-%s'" index-webfontconfig-js-hash)
                                  ;; inline script in index.html that loads Google Analytics
-                                 "'sha256-uKEj/Qp9AmQA2Xv83bZX9mNVV2VWZteZjIsVNVzLkA0='"]
+                                 (format "'sha256-%s'" index-ganalytics-js-hash)
+                                 ;; inline script in init.html
+                                 (format "'sha256-%s'" init-js-hash)]
```

Comment:
Then if `inline-js-hashes` is a single var you can calculate `script-src` like this:

```clj
(concat
 ["'self'"
  "'unsafe-eval'"                   
  "https://maps.google.com"
  "https://apis.google.com"
  "https://www.google-analytics.com"
  "https://*.googleapis.com"
  "*.gstatic.com"
  (when config/is-dev?
    "localhost:8080")]
 (map (partial format "'sha256-%s'")  inline-js-hashes))
```

Reply:
So you eliminate all the repetitive calls to `format`

Reply:
Done.